court/tests.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from court.models import Court
+from decimal import Decimal
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+
+
+@pytest.mark.django_db
+class TestCourtModel:
+
+    @pytest.fixture
+    def court(self):
+        court = Court.objects.create(courtID=1, x=Decimal('11'), y=Decimal('22'),
+                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+        return court
+
+    def test_validate_instance(self, court):
```

Comment:
```suggestion
    def test_get_instance(self, court):
```
You have another test case in which you validate the instance.